template/table.py

```diff
@@ -180,7 +180,6 @@ def setPageRecord(self, record, recordLocation):
         self.physicalPages[RID_COLUMN].write(RID)
         self.physicalPages[TIMESTAMP_COLUMN].write(record.timestamp)
         self.physicalPages[SCHEMA_ENCODING_COLUMN].write(record.encoding)
-        self.physicalPages[KEY_COLUMN].write(record.key)
```

Comment:
the column representing the key is one of the data columns; hopefully that doesn't affect the change to PhysicalPages here